SignalServiceKit/src/Devices/OWSLinkedDeviceReadReceipt.m

```diff
@@ -0,0 +1,53 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSLinkedDeviceReadReceipt.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation OWSLinkedDeviceReadReceipt
```

Comment:
Note: this class got renamed and I didn't bother with a migration since we'll quickly recover from lost read receipts.

Reply:
👍